FrontEnd/src/components/ProfileDetail/MainInfo/ProfileDetailNavBar.jsx

```diff
@@ -0,0 +1,48 @@
+import { HashLink } from 'react-router-hash-link';
+import classes from './ProfileDetailNavBar.module.css';
+
+function ProfileDetailNavBar() {
+  const menuItems = [
+    'Про компанію',
+    'Стартап',
+    'Товари / послуги',
+    'Логістика товарів / послуг',
+    'Формат співпраці',
+  ];
+  const menuLinks = [
+    'about-company',
+    'startup',
+    'products-services',
+    'logistics',
+    'cooperation',
+  ];
+
```

Comment:
I think it can be organized as an array of objects. Also it may be a good idea to make it a constant outside of the component.

Reply:
Totally agree

Reply:
Done